specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/stable/2016-05-01/Alert.json

```diff
@@ -88,6 +97,9 @@
                         "schema": {
                             "$ref": "#/definitions/Alert"
                         }
+                    },
+                    "404": {
+                        "description": "NOT FOUND"
                     }
```

Comment:
Same here, unsuccessful responses should not be included

Reply:
The team prefers not to throw an error on 404 and instead return null.